avm/utilities/pipelines/staticValidation/psrule/ps-rule.yaml

```diff
@@ -81,3 +81,4 @@ rule:
     - Azure.VM.UseHybridUseBenefit
     - Azure.AppConfig.PurgeProtect
     - Azure.APIM.MultiRegion # Team agreed this is too expensive for most use cases and is safe to ignore.
+    - Azure.MySQL.MaintenanceWindow # https://msdata.visualstudio.com/Database%20Systems/_workitems/edit/2788114
```

Comment:
Link doesn't work for me.
Regardless, I recommend to pad it a bit like '# Must be excluded until <link> is fixed. Feature is currently not deployable'

Reply:
@AlexanderSehr Updated